shared/chat/conversation/container.js

```diff
@@ -73,7 +73,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     } else {
       type = 'normal'
     }
-  } else if (stateProps._pendingConversationUsers) {
+  } else if (stateProps._pendingConversationUsers && stateProps._pendingConversationUsers.size) {
```

Comment:
In the case where the team was the only chat in your inbox, this was causing the wrong conversation view to show since `_pendingConversation` is an empty set which is truthy.